backend/chainlit/langchain/callbacks.py

```diff
@@ -350,7 +213,14 @@ def _build_llm_settings(
         # make sure there is no api key specification
         settings = {k: v for k, v in merged.items() if not k.endswith("_api_key")}
 
-        return provider, settings
+        model_keys = ["model", "model_name", "deployment", "deployment_name"]
```

Comment:
correct order is             model_keys = ["azure_deployment", "deployment_name", "model", "model_name"]